requirements.in

```diff
@@ -2,7 +2,10 @@ sphinx==1.7.5
 sphinx-intl
 transifex-client
 sphinx_rtd_theme
+CommonMark==0.7.5
```

Comment:
Can we have a comment explaining why this is frozen? I can just see it as puzzling when someone comes back to it much later.
At some point we'll have to do the upgrade too (and fix all the imports in places), so it's good to have a comment to guide that.

Reply:
Added a comment: "Higher versions rename the package which mean we need code changes before we can upgrade, see https://github.com/rtfd/CommonMark-py/blob/master/CHANGELOG.md#080-2018-09-03" - is that okay?